WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2414,9 +2438,39 @@ static GSourceFuncs _handlerIntervention =
 
             WKPageGroupSetPreferences(pageGroup, preferences);
 
+            auto userContentController = WKUserContentControllerCreate();
             auto pageConfiguration = WKPageConfigurationCreate();
             WKPageConfigurationSetContext(pageConfiguration, wkContext);
             WKPageConfigurationSetPageGroup(pageConfiguration, pageGroup);
+            WKPageConfigurationSetUserContentController(pageConfiguration, userContentController);
+
+            if (_config.UserScripts.IsSet()) {
```

Comment:
As the code is similar to the above put it in a private function and call it in both places perhaps?